content/policies/index.mdx

```diff
@@ -1,8 +1,9 @@
 ---
-policies
+title: Policies
 ---
 
-These are updated from time to time.  Their sources are stored in a git
-repository at [https://github.com/npm/documentation/content/policies](https://github.com/npm/documentation/content/policies).
+These are the legal policies of npm, Inc.
```

Comment:
@MylesBorins I don't know if it should be referenced as `npm, Inc.` or `npm`. The rest of the policy pages use `npm, Inc.`, so I kept it for consistency